vignettes/articles/oauth.Rmd

```diff
@@ -37,24 +37,28 @@ That means as a programmer you never want to touch user name-password pairs beca
 
 Avoiding that problem lead to the creation of OAuth.
 The basic idea is that instead of your package asking the user to give you their user name and password, you instead ask them to log in and give your package permission to use the API on their behalf.
-The API gives this permission in the form of an **access token**, which is essentially a random string of numbers and letters.
-An access token tends to be very short lived, lasting maybe only days or hours, and is bound to a specified **scope** of access.
+The API gives this permission in the form of an **access token** which is essentially a random string of numbers and letters, e.g.
+`UfNlXaEog03hdRPTUPpEInEiIW01jI1WcjOB`.
+An access token is very short lived, lasting maybe only days, and is bound to a specified **scope** of access.
 The access token has some big advantages over a user name and password:
 
 -   It's short-lived so if it's lost or stolen, there's only a limited amount of time where it can be abused.
 -   It has limited scope, so even if stolen, it can't be used to do something particularly nefarious like changing your password or contact details.
 -   It's bound to a specific application, so it can be invalidated (cancelled) without affecting any other uses.
 
-If you have an access token you can use it to authenticate an API by passing a **bearer token** in the `Authorization` header, which you can do with `req_auth_bearer_token()`.
+If you have an access token you can use it to authenticate an API by passing it as a **bearer token** in the `Authorization` header, which you can do with `req_auth_bearer_token()`.
 However, in most cases you will want to let httr2 manage this by calling one of the `req_oauth_` functions we'll talk about shortly.
 
 One of the reasons that you want httr2 to manage your tokens is that because access tokens are so short lived, they're often accompanied by a **refresh token**.
 A refresh token lasts for a longer amount of time and has one job: it allows you to get a new access token when the previous one expires.
-This leads to a hierarchy of credentials from strongest to weakest:
+You need to look after an refresh token a little more carefully than an access token.
+In particular, you should never include a refresh token in a HTTP request as that's the job of the access token.
 
--   user name + password: your code should never touch these!
--   refresh token: it's okay to store this on your computer.
--   access token: you submit this with every request.
+Overall this leads to a hierarchy of credentials from weakest to strongest:
+
+-   The access token only lasts a couple hours and needs to be submitted this with every request.
```

Comment:
```suggestion
-   The access token usually only lasts a couple hours and needs to be submitted with every request.
```

Some APIs are bad about the hierarchy and essentially treat access tokens like refresh tokens.